packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx

```diff
@@ -46,6 +46,7 @@ export interface FileDropzoneProps {
    */
   fileListRenderer?: (file: DropzoneFile, removeFile: (file: DropzoneFile) => void) => ReactNode;
   onFileRemove?: (file: DropzoneFile) => void;
+  primaryTextSupplier?: (files?: DropzoneFile[], options?: BackwardsCompatibleDropzoneOptions) => string;
```

Comment:
Is there a reason to make the `files` parameter optional? As far as I can see the component sets it to an empty array by default.